README.md

```diff
@@ -49,7 +49,7 @@ A note about the gamer method. If you already know the gamers xuid you can use t
 | Method | Value | Optional | Short Description |
 |---       |---    |---              |---       |
 | `get(method=<string>)`        | string  | `term=<string>` | API calls. |
-| `send_message(message=<string>, xuids=[])` | string, list | n/a | Send a message to the following xuids |
+| `send_message(message=<string>, gamers=[])` | string, list | n/a | Send a message to the following xuids |
```

Comment:
Good call.  Updated in https://github.com/xboxapi/Python-Wrapper/pull/18/commits/9dc47b888f9bef46def715a60f8a6c4be19b0a64.